### PR TITLE
Learn modular update

### PR DESCRIPTION
**Pull request by alex-beckett**
- Add links to corresponding glossary pages for certain words in learn modular articles
- Add twitter metadata to learn modular pages so that when linked on twitter they display image cards

egsdfa

### CODE REVIEW

**Comment by YazzyYaz** [comment]
From a high-level, looks good. Is there a Gatsby link I can use to view the changes?

**Comment by alex-beckett** [comment]
[Here](https://celestiaorglearnmodularupdate.gatsbyjs.io/)

File: src/pages/markdown-pages/learn/basics-of-modular-blockchains.md [hunk 1]
```diff
@@ -75,3 +73,10 @@ Modular blockchains aren’t constrained by having to handle all the functions.
 2. Monolithic blockchains are limited by handling all functions on a single layer. To increase throughput, security or decentralization must be sacrificed.
 3. Modular blockchains provide many benefits over their monolithic counterparts, some of which include efficiently creating new blockchains, sovereignty, and scalability.
 </div>
+
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@CelestiaOrg">
+<meta name="twitter:creator" content="@likebeckett">
+<meta name="twitter:title" content="Modular Scalability">
+<meta name="twitter:description" content="Blockchain researchers have long grappled with the challenge of creating the optimal system. Many architectures have been tried whose goal was to accommodate all users on a single chain or a tightly coupled group of chains that live under a single network.">
+<meta name="twitter:image" content="https://github.com/celestiaorg/celestia.org/blob/main/src/pages/markdown-pages/learn/images/learn-modular-twitter-card.png">
```
Comment (YazzyYaz) [inline src/pages/markdown-pages/learn/basics-of-modular-blockchains.md]: meta labels must be in `<head></head>`

File: src/pages/markdown-pages/learn/creating-new-blockchains.md [hunk 2]
```diff
@@ -54,3 +54,10 @@ Coupled with a data availability layer, execution layers can conduct unconstrain
...
 3. The execution environments that execution layers can build are limited by being deployed on a settlement layer. For an execution layer deployed on a data availability layer, there are no restraints on the type of execution environment that can be built and deployed. This facilitates experimentation and innovation in the execution environment that is unconstrained by any dependency on a data availability layer.  
 </div>
+
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@CelestiaOrg">
+<meta name="twitter:creator" content="@likebeckett">
+<meta name="twitter:title" content="Modular Scalability">
+<meta name="twitter:description" content="Traditionally, creating a new blockchain had been a resource-intensive endeavor, imposing considerable time and monetary costs. Although blockchains can be implemented more efficiently now, existing hurdles are still present that cause friction in the bootstrapping process.">
+<meta name="twitter:image" content="https://github.com/celestiaorg/celestia.org/blob/main/src/pages/markdown-pages/learn/images/learn-modular-twitter-card.png">
```
Comment (YazzyYaz) [inline src/pages/markdown-pages/learn/creating-new-blockchains.md]: Ditto

File: src/pages/markdown-pages/learn/modular-architectures.md [hunk 3]
```diff
@@ -70,3 +70,9 @@ Since there is no settlement layer involved, only the data availability layer is
 4. A modular stack can also consist of an execution layer that runs natively on a consensus and data availability layer. Under this construction, execution layer nodes would verify blocks through its peer-to-peer layer rather than from a settlement layer contract. This enables sovereignty for the execution layer as it can determine its environment without permission from any underlying layers.
 </div>
 
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@CelestiaOrg">
+<meta name="twitter:creator" content="@likebeckett">
+<meta name="twitter:title" content="Modular Scalability">
+<meta name="twitter:description" content="A new paradigm is emerging in which modular blockchains are enabling new chains to be constructed in ways that were not previously possible.">
+<meta name="twitter:image" content="https://github.com/celestiaorg/celestia.org/blob/main/src/pages/markdown-pages/learn/images/learn-modular-twitter-card.png">
```
Comment (YazzyYaz) [inline src/pages/markdown-pages/learn/modular-architectures.md]: Also here

File: src/pages/markdown-pages/learn/modular-scalability.md [hunk 4]
```diff
@@ -71,3 +71,11 @@ The vast majority of the settlement layers transaction activity should consist o
 2. Data availability layers scale block verification and minimize bandwidth costs for light nodes. This is possible with data availability sampling, which allows light nodes to verify data availability by only downloading small random samples of a block.
 3. Settlement layers can scale using the same techniques from execution and data availability layers. In their current form, settlement layers are unideal. A more optimal approach would be to create an environment that limits smart contracts to favor interactions with execution layers.
 </div>
+
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@CelestiaOrg">
+<meta name="twitter:creator" content="@likebeckett">
+<meta name="twitter:title" content="Modular Scalability">
+<meta name="twitter:description" content="Monolithic blockchains have pursued varying scalability techniques over time, many of which have proven unsuccessful. Modular blockchains present a way for a monolithic blockchain to be split up among multiple specialized layers.">
```
Comment (YazzyYaz) [inline src/pages/markdown-pages/learn/modular-scalability.md]: Also here

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

**Comment by alex-beckett** [comment]
All meta tags have been moved into the <head>

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**